library/src/error/flatten/flatten.ts

```diff
@@ -136,15 +134,18 @@ export function flatten<TSchema extends BaseSchema | BaseSchemaAsync = any>(
 
 /**
  * Flatten the error messages of issues.
- *
  * @param issues The issues.
- *
  * @returns Flat errors.
  */
 export function flatten<TSchema extends BaseSchema | BaseSchemaAsync = any>(
   issues: Issues
 ): FlatErrors<TSchema>;
 
+/**
+ *
```

Comment:
I prefer not to have a JSDoc comment for the function overload implementation if the overload definitions are already commented. Can we control this globally? Otherwise we should disable this rule individually as needed.

Reply:
> not to have a JSDoc comment for the function overload implementation if the overload definitions are already commented. 

this plugin is implemented as you said, in other words, we have to write JSDoc only in the implement section, and we do not have to write in the overload sections.

However, there are many JSDocs in the overload sections in Valibot, and this plugin causes errors.

So I deleted the JSDocs in the overload sections just like the JSDocs of  the overload section of [brand.ts](https://github.com/kazizi55/valibot/blob/76ffe2ecbd1b8ae80fead75fb3a18a6cb7823aee/library/src/methods/brand/brand.ts#L1-L698) has been deleted already, so that the errors won't occur.

https://github.com/fabian-hiller/valibot/pull/205/commits/76ffe2ecbd1b8ae80fead75fb3a18a6cb7823aee

Reply:
Thank you! I will review it in the coming days.